app/models/users/setting.rb

```diff
@@ -18,7 +19,19 @@ class Setting < ApplicationRecord
     validates :feed_referential_link, inclusion: { in: [true, false] }
     validates :feed_url, length: { maximum: 500 }, allow_nil: true
     validates :inbox_guidelines, length: { maximum: 250 }, allow_nil: true
-  end
 
-  # TODO: @msarit Re-add feed_url validation after updates are pointed directly to users_settings table
+    validate :validate_feed_url, if: :feed_url_changed?
+
+    private
+
+    def validate_feed_url
```

Comment:
This method used to live in `user.rb`; moved it over to `setting.rb` now that `feed-url` is in `users_settings`